README.md

```diff
@@ -1 +1,37 @@
-# INF 6420 - Intro to Web Development
+# Angie Moore's INF6420 Project 
+
+A booklist and reccomendation site.
+
+## Wireframes 
+
+I struggle with organizaton and spatial awareness, so these wireframes appear to be the best to keep my content streamlined and organized
+
+![Wireframe of Home Page](wireframes/wireframe1.jpeg)
```

Comment:
Hi @hd5420 the image locations need to match the name of the folder. They are currently not displaying (See: https://github.com/INF6420/ss22-hd5420/blob/e44c1fb6323a1b2496ba9f2b982a0d91545bc60e/README.md). You have a capital W for the folder name, but a lowercase w for the image source in the readme file. If you rename the folder to use a lowercase w, the images will display correctly.

Reply:
I updated the folder name on my desktop, but nothing is changing in GitHub. I'm not sure what to do to fix it.